addons/l10n_it_edi/tests/test_ir_mail_server.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+from unittest.mock import patch
+from collections import namedtuple
+
+from odoo.modules.module import get_module_resource
+from odoo.tests import common
+from odoo import tools
+
+_logger = logging.getLogger(__name__)
+
+
+class MailServerTests(common.TransactionCase):
```

Comment:
Use `AccountTestInvoicingCommon` to avoid depending on the demo data and to avoid running the test on a company not compatible with `l10n_it_edi`

Reply:
Done. I've used this which is even more specific:
```py
from odoo.addons.account_edi.tests.common import AccountEdiTestCommon
```
It's caused me some trouble, because it's supposed to work with "accountman" user, but the fetchmail server has to be sudoed.

Reply:
Please have a look at the entire test file, it will be easier because I've turned it around a bit. The logic is pretty much the same, though.